### PR TITLE
Update copy

### PR DESCRIPTION
**Pull request by chipit24**
Related issues:
 * https://github.com/ProcessMaker/screen-builder/issues/567
 * https://github.com/ProcessMaker/screen-builder/issues/569

Screen Shot 2020-02-25 at 2 06 22 PM

Screen Shot 2020-02-25 at 2 15 26 PM

### CODE REVIEW

File: resources/js/processes/screen-builder/screen.vue [hunk 1]
```diff
@@ -277,7 +277,7 @@ import formTypes from "./formTypes";
         this.validationWarnings.splice(0);
 
         if (this.type === formTypes.form && !this.containsSubmitButton()) {
-          this.validationWarnings.push(this.$t('Form requires a submit button to be saved'));
+          this.validationWarnings.push('Warning: Screens without save buttons cannot be executed.');
```
Comment (chipit24) [inline resources/js/processes/screen-builder/screen.vue]: I removed the `$t` translator function as it truncates text before and including the colon (`:`). There also didn't appear to be an entry for this in the `en.json` file.

**Comment by velkymx** [comment]
@chipit24 looks like this broke a test.

**Comment by chipit24** [comment]
@velkymx Would you know why these tests are failing on CI?

When run locally, unit tests are failing both for develop and my branch. However, my branch has one additional failing unit test:
```
3) Tests\Feature\Api\PerformanceTest::testRoutesSpeed with data set #13 ('api.processes.start', array())
Slow route response [api.processes.start]
             Speed ~ 2 [reqs/sec]
Failed asserting that 0.09300792841114154 is equal to 0.1 or is greater than 0.1.

/home/vagrant/processmaker/tests/Feature/Api/PerformanceTest.php:232
```

But all I'm doing in this branch is changing some text, and the text that I've changed doesn't appear to be referenced in any tests. To make things more confusing, the test that's failing above seems to be passing on CI; circleCI is reporting something different than phpunit locally.

**Comment by chipit24** [comment]
I've broken out the two text changed in this PR into two separate PRs:
 * https://github.com/ProcessMaker/processmaker/pull/2903
 * https://github.com/ProcessMaker/processmaker/pull/2902

Hopefully those two PRs will help better isolate the reason for the phpunit test failures. Closing this PR in favour of those.

**Pull request closed without merging**